Moya.podspec

```diff
@@ -31,7 +31,7 @@ Pod::Spec.new do |s|
   s.subspec "ReactiveCocoa" do |ss|
     ss.source_files = "Moya/ReactiveCocoa/*.swift"
     ss.dependency "Moya/ReactiveCore"
-    ss.dependency "ReactiveCocoa", "~> 4.0-alpha.1"
+    ss.dependency "ReactiveCocoa", "= 4.0.0-alpha-3"
```

Comment:
Two things..
1) as you say, its possible/likely 4th alpha will have breaking changes
2) I tested `~> 4.0.0-alpha-3` and it resolves to `4.0.4-alpha-1`.. I guess CP prioritises the version number and ignore the alpha bit. as mentioned above, have opened https://github.com/CocoaPods/Specs/pull/13588 to change `4.0.0-alpha-1` to the contents of `4.0.4-alpha-1`, and kill `4.0.(1/2/3/4)-alpha-1`